src/test/resources/network/brightspots/rcv/test_data/aliases_cdf_json/aliases_cdf_json_config.json

```diff
@@ -0,0 +1,68 @@
+{
+  "tabulatorVersion": "TEST",
```

Comment:
> Thanks for catching the accidentally-updated test -- perhaps we should stop the UI from forcing you to re-save a config if the version is "TEST"? It's a pain to constantly revert those. I can make a ticket for that if you think it'd help y'all as well.

Yeah, I think that's a good idea, please do file an issue for that.

Reply:
https://github.com/BrightSpots/rcv/issues/664